Node/core/tests/localization.js

```diff
@@ -1,11 +1,22 @@
 var assert = require('assert');
 var builder = require('../');
 
+
+
 describe('localization', function() {
     this.timeout(5000);
+
```

Comment:
There were errors with these tests not being able to find the test data. This fixes that.